plugin/storage/integration/kafka_test.go

```diff
@@ -119,8 +121,13 @@ func (*ingester) FindTraceIDs(context.Context, *spanstore.TraceQueryParameters)
 }
 
 func TestKafkaStorage(t *testing.T) {
+	defer testutils.VerifyGoLeaksOnce(t)
 	SkipUnlessEnv(t, "kafka")
 	s := &KafkaIntegrationTestSuite{}
-	s.initialize(t)
+	f, c := s.initialize(t)
```

Comment:
once again, we don't structure the tests to return something that the caller needs to manually close, we use t.Cleanup for that